src/System.Net.Http/src/System/Net/Http/Managed/ValueStringBuilder.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text
+{
+    internal struct ValueStringBuilder
```

Comment:
Do we not have any generic char array builder type that could do this job? It seems very generic.